### ISSUE TITLE
Make turtles consume fuel

### ISSUE THREAD

**Issue by ghost** [BiLachner/TurtleMiner#9]
I suggest adding option of making turtles consume fuel, such as coal. Each action could cost a certain percentage of the fuel, after which the user would have to add more. This would make it a bit better balanced IMO for non-creative worlds. Turtles could even autonomously refuel themselves maybe by digging coal blocks.

**Comment by octacian** [comment]
Now that turtles have inventories, this feature will be coming soon. I'm unsure as to whether I'm going to implement this or a text editor first. Turtles would have already had a text editor by now today, as I had computers mod that I was going to merge turtles in to. But, my HDD failed 2 days ago _just_ before the rewrite of the entire mod was pushed.

What I'm saying, is yeah, this is a planned feature, but unless @BiLachner does it, sadly it won't happen too soon.

**Comment by BiLachner** [comment]
I agree with this idea that was already mentioned in the forum. For me only important is that we should be able to deactivate the features to have simpler turtles. So, go on ;-)

**Comment by octacian** [comment]
OK, will do ASAP. If you get around to it, let me know. Still recovering from HDD failure as I mentioned.

Down to low-level recovering methods, so it'll take some time.
